appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockLabel.java

```diff
@@ -100,9 +100,18 @@ private void setFontTypefaceProperty(String text) {
   private void setTextProperty(String text) {
     savedText = text;
     if (getPropertyValue(PROPERTY_NAME_HTMLFORMAT).equals("True")) {
-      labelWidget.setHTML(SimpleHtmlSanitizer.sanitizeHtml(text).asString());
+      String sanitizedText = SimpleHtmlSanitizer.sanitizeHtml(text).asString();
+      labelWidget.setHTML(
+        (sanitizedText == null) ? 
+        sanitizedText : 
+        sanitizedText.replaceAll("\\\\n", "<br>")
+      );
     } else {
-      labelWidget.setText(text);
+      labelWidget.setText(
+        (text == null) ?
+        text :
+        text.replaceAll("\\\\n", "<br>")
```

Comment:
I think you have too many slashes here. We want to replace the string "\n" with "<br>", so you only need one pair of `\\` to capture the single `\` before the `n`.

Reply:
@ewpatton Are you suggesting that it should be `text.replaceAll("\\n", "")`?

Reply:
Yes I believe that should work.

Reply:
@ewpatton Tried that, but it did not change `\n` to actual line breaks in the designer.

Reply:
Oh, sorry I thought I was replying to the other one. For this, you need to switch to `<br>`, but you'll also need to call setHTML rather than setText because otherwise the `<br>` will just be rendered as such rather than an actual line break.

Reply:
No worries! I'll try that.